tools/topology/topology2/include/components/igo_nr.conf

```diff
@@ -0,0 +1,91 @@
+#
+#
+# A generic IGO_NR component. All attributes defined herein are namespaced
```

Comment:
I suggest add full name of IGO_NR in a place, let everyone know what exactly IGO_NR is.

Reply:
@singalsu , are you adding full name for IGO_NR?

Reply:
Yep, I will update this simple PR next. Don't approve yet.